src/Schema/Teradata/TeradataSchemaReflection.php

```diff
@@ -34,7 +34,9 @@ public function getTablesNames(): array
 EOT
         );
 
-        return DataHelper::extractByKey($tables, 'TableName');
+        /** @var string[] $extracted */
+        $extracted = DataHelper::extractByKey($tables, 'TableName');
```

Comment:
jsi to chtel uplne zrusit ten DataHelper ne?

Reply:
Nechtěl sem do toho tady už víc zabrušovat, ale odstranění je tady https://github.com/keboola/php-table-backend-utils/pull/60

Reply:
ok